masterlist.yaml

```
@@ -4217,11 +4217,24 @@ plugins:
       - crc: 0x75D6B87E
         util: 'TES5Edit v4.0.2j'
   - name: 'Bob The Merchant.esp'
-    url: [ 'https://www.nexusmods.com/skyrim/mods/22304' ]
+    url:
+      - link: 'https://www.nexusmods.com/skyrim/mods/22304/'
+        name: 'The Bob Trading Company'
+    msg:
+      - <<: *obsolete
+        condition: 'checksum("Bob The Merchant.esp",05F9D7C3)'
```

Missing a space :P :

```suggestion
        condition: 'checksum("Bob The Merchant.esp", 05F9D7C3)'
```

---

I've added it, though the Skyrim masterlist seems to be pretty inconsistent in this regard. There are multile instances of checksum checks with and without that space.

---

Yeah, probably because of the conversion from BOSS IIUC. It'd be good to match our current practices going forward though.